packages/cli/subtasks/check-initializable.js

```diff
@@ -0,0 +1,31 @@
+const { subtask } = require('hardhat/config');
+const { SUBTASK_CHECK_INITIALIZATION } = require('../task-names');
+const logger = require('@synthetixio/core-js/utils/io/logger');
+const { capitalize } = require('@synthetixio/core-js/utils/misc/strings');
+
+subtask(
+  SUBTASK_CHECK_INITIALIZATION,
+  'Check if current contract is initializable and if it is initialized'
+).setAction(async (taskArguments, hre) => {
+  const target = hre.deployer.deployment.general.contracts[hre.cli.contractName];
+  const address = target.proxyAddress || target.deployedAddress;
+  const abi = hre.deployer.deployment.abis[hre.cli.contractName];
+  const abiFunctions = abi.filter((abiItem) => abiItem.name && abiItem.type === 'function');
+
+  const capitalizedContractName = capitalize(hre.cli.contractName.split(':')[1]);
+  const isInitializableFunctionName = `is${capitalizedContractName}Initialized`;
```

Comment:
Small semantic improvement: this should be called isInitializedFunctionName. Its existance determines if the contract is initializable. Becomes more relevant when you actually call the function a few lines below.

Reply:
in fact, is not semantic, it was the wrong name (a mental typo) :)